config/Config-template.groovy

```diff
@@ -237,12 +239,30 @@ environments {
     RModules.imageURL = "/tempImages/" //must end and start with /
 
     production {
-        // The working direcotry for R scripts, where the jobs get created and
+        // The working directory for R scripts, where the jobs get created and
         // output files get generated
         RModules.tempFolderDirectory = jobsDirectory
+
+        // Whether to copy the images from the jobs directory to another
+        // directory from which they can be served. Should be false for
+        // performance reasons. Files will be served from the
+        // tempFolderDirectory instead, which should be exposed as
+        // <context path>/analysisFiles (formerly: <context path>/tempImages)
+        RModules.transferImageFile = false
+
+        // Copy inside the exploded WAR. In actual production, we don't want this
+        // The web server should be able to serve static files from this
+        // directory via the logical name specified in the imageUrl config entry
+        // Not needed because transferImageFile is false
+        //Rmodules.temporaryImageFolder = explodedWarDir + '/images/tempImages/'
```

Comment:
You should not have reintroduced these two options. They are not used anymore, as the files are sent directly from their location with sendfile.